enhancements/update/change-management-and-maintenance-schedules.md

```diff
@@ -0,0 +1,1158 @@
+1. The HCP cluster has a large number of worker nodes and draining and rebooting them is time-consuming. 
+1. At 23:59 UTC Saturday night, 80% of worker-nodes have been updated. Since the maintenance schedule still permits the initiation of material changes, another worker-node begins to be updated.
+1. The update of this worker-node continues, but at 00:00 UTC Sunday, no further material changes are permitted by the change management policy and the worker-node update process is effectively paused.
+1. Because not all worker-nodes have been updated, changes are still reported as pending via metrics for NodePool.  **TODO: Review with HyperShift. Pausing progress should be possible, but a metric indicating changes still pending may not since they interact only through CAPI.**
+1. The HCP cluster runs with worker-nodes at mixed versions throughout the month. The N-1 skew between the old kubelet versions and control-plane is supported.
+1. **TODO: Review with Service Delivery. If the user requested another minor bump to their control-plane, how does OCM prevent unsupported version skew today?**
+1. On the next first Saturday, the worker-nodes updates are completed.
+
+#### OCM Standalone Standard Change Management Scenario
+
+1. User interactions with OCM to configure a maintenance schedule are identical to [OCM HCP Standard Change Management Scenario](#ocm-hcp-standard-change-management-scenario).
+   This scenario differs after OCM accepts the maintenance schedule configuration. Control-plane updates are permitted to be initiated to any Saturday UTC.
+   Worker-nodes must wait until the first Saturday of the month.
+1. OCM (through various layers) configures the ClusterVersion and worker MachineConfigPool(s) (MCP) for the cluster with appropriate `changeManagement` stanzas.
+1. Company workloads are added to the new cluster and the cluster provides value.
+1. To leverage a new feature in OpenShift, the service consumer plans to update the minor version of the platform.
+1. Via OCM, the service consumer requests the minor version update. They can do this at any time with confidence that the maintenance 
+   schedule will be honored. They do so on Wednesday.
+1. OCM (through various layers) updates the ClusterVersion resource on the cluster indicating the new release payload in `desiredUpdate`.
+1. The Cluster Version Operator (CVO) detects that its `changeManagement` stanza does not permit the initiation of the change.
+1. The CVO sets a metric indicating that changes are pending for ClusterVersion. Irrespective of pending changes, the CVO also exposes a 
+   metric indicating the number of seconds until the next window in which material changes can be initiated.
+1. Since MachineConfigs do not match in the desired update and the current manifests, the CVO also sets a metric indicating that MachineConfig
```

Comment:
How does this work? Do we need code in CVO to compare MachineConfigs? Or would CVO ask MCO to do the comparison somehow and report the results? Or can we figure out that a change is needed without deep semantic comparison of the manifests and live state of the system?

Reply:
I actually think it is safe to _assume_ it will change. If CVO has not reconciled manifests for the desired release payload, just set this metric. This is true in 100% of non-hotfix release  payloads since RHCOS is changing. Additionally, the price of being wrong is very low. The master changes will roll out and the apparently pending machine changes quickly disappear without any disruption.

Reply:
That seems like a reasonable assumption. Is that captured in the body somewhere?

Reply:
Added in latest push.